src/components/Checkbox.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+interface Props {
+  checked: boolean;
+  onChange: () => void;
+}
+
+const Checkbox: React.FC<Props> = ({checked, onChange}) => {
```

Comment:
We have 2 types of checkbox, one for table and one for the form. Maybe have different names for them?